plot_features.py

```diff
@@ -138,34 +148,41 @@ def plot_features_file(featurefile, mapInfo): #{{{
     for feature in featuredat['features']:
         polytype = feature['geometry']['type']
         coords = feature['geometry']['coordinates']
-        feature = feature['properties']['name']
-        print '  feature: %s'%feature
+        featurename = feature['properties']['name']
+        print '  feature: %s'%featurename
 
         color_num = feature_num % len(colors)
         marker_num = feature_num % len(markers)
 
         try:
+            # basically should be able to use descartes to plot polygon generally
             if polytype == 'MultiPolygon':
+                # update not implemented
                 for poly in coords:
                     for shape in poly:
                         plot_poly(mapInfo,shape,colors[color_num])
             elif polytype == 'Polygon' or polytype == 'MultiLineString':
-                for poly in coords:
-                    plot_poly(mapInfo,poly,colors[color_num])
+                try:
+                    patch = feature['geometry']
+                    plot_poly(mapInfo, patch, colors[color_num])
+                except:
+                    for poly in coords:
+                        plot_poly(mapInfo, poly, colors[color_num])
             elif polytype == 'LineString':
-                plot_poly(mapInfo,coords,colors[color_num],filled=False)
+                plot_poly(mapInfo, coords, colors[color_num], filled=False)
```

Comment:
I realize this is really outside the scope of this PR but could you also make LineStrings get plotted as lines instead of polygons?  I believe even plotting them as unfilled polygons is wrong since they are sometimes (often?) transects, which are not closed shapes like polygons.